batch/2.0/_index.md

```diff
@@ -36,19 +36,20 @@ Please reference that ballot for the official results.
 
 ## Release Review
 
-The Specification Committee Ballot concluded successfully on 2020-mm-dd with the following results.
+The Specification Committee Ballot concluded successfully on 2020-08-27 with the following results.
 
 | Representative                                 | Representative for: | Vote |
 |------------------------------------------------|---------------------|------|
-| Kenji Kazumura, Michael DeNicola               | Fujitsu             |      |
-| Dan Bandera, Kevin Sutter                      | IBM                 |      |
-| Bill Shannon, Ed Bratt                         | Oracle              |      |
-| Mark Wareham, Steve Millidge                   | Payara              |      |
-| Scott Stark, Mark Little                       | Red Hat             |      |
-| David Blevins, Cesar Hernandez                 | Tomitribe           |      |
-| Ivar Grimstad                                  | EE4J PMC            |      |
-| Alex Theedom                                   | Participant Members |      |
-| Werner Keil                                    | Committer Members   |      |
-|                                                | Total               |      |
-
-The ballot was run in the [jakarta.ee-spec mailing list]()
+| Kenji Kazumura                                 | Fujitsu             |  1   |
+| Dan Bandera, Kevin Sutter                      | IBM                 |  1   |
+| Ed Bratt, Dmitry Kernilov                      | Oracle              |  1   |
+| Andrew Pielage, Matt Gill                      | Payara              |  1   |
+| Scott Stark, Mark Little                       | Red Hat             |  1   |
+| David Blevins, Jean-Louis Monteiro, Cesar Hernandez  | Tomitribe           |  1   |
+| Ivar Grimstad                                  | EE4J PMC            |  1   |
+| Marcelo Ancelmo, Martijn Verburg               | Participant Members |  1   |
+| Werner Keil                                    | Committer Members   |  1   |
+| Scott (Congquan) Wang                          | Enterprise Members  |      |
```

Comment:
Per the discussion on the mailing list, we should not leave this blank.  Either "no vote cast" or "no vote".  And, did you ping @ScottWangNL directly with a reminder about this vote?

Reply:
Cast votes should be recorded as +1, 0, or -1. If we do that, then a blank cell would mean "no vote", so we won't need to specify that...

Reply:
Even with the +1, -1, 0 convention, I still think a "no vote" would be clearer than blank.

Reply:
> Per the discussion on the mailing list, we should not leave this blank. Either "no vote cast" or "no vote". And, did you ping @ScottWangNL directly with a reminder about this vote?

I put "no vote cast" for the time being.
After I get the response from @ScottWangNL, I will update.